qa/test_template.sh

```diff
@@ -8,8 +8,13 @@ set -x
 topdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/..
 source $topdir/qa/setup_test_common.sh
 
+# Set runner for python tests
+python_test_runner_package="nose"
+python_test_runner="python -m nose"
+python_test_args="--verbose -s"
+
 # Set proper CUDA version for packages, like MXNet, requiring it
-pip_packages=$(echo ${pip_packages} | sed "s/##CUDA_VERSION##/${CUDA_VERSION}/")
+pip_packages=$(eval "echo \"${pip_packages}\"" | sed "s/##CUDA_VERSION##/${CUDA_VERSION}/")
```

Comment:
This change allows to evaluate `pip_packages` and replace placeholders with current values. 

For example:

`${python_test_runner_package} numpy>=1.17 numba scipy librosa==0.8.1` 

becomes:

`nose numpy>=1.17 numba scipy librosa==0.8.1`